aws/cloudformation/lambdas/marketing-router/index.js

```diff
@@ -7,6 +7,254 @@ const marketingDomain = '${InternalMarketingDomainName}'
+  "/en-US/students/scholarships": true,
+  "/en-US/students/internships-and-apprenticeships": true,
+  "/en-US/curriculum/maker": true,
+  "/en-US/students/online-courses": true,
+  "/en-US/districts/partners": true,
+  // P-0-2
+  "/athome": true,
+  "/educate/resources/videos": true,
+  "/curriculum/csd": true,
+  "/careers-with-cs/index": true,
+  "/curriculum/middle-school": true,
+  "/ai/teaching-assistant": true,
+  "/about/careers": true,
+  "/curriculum/coding-with-ai": true,
+  "/curriculum/game-design": true,
+  "/curriculum/how-ai-works": true,
+  "/diversity": true,
+  "/curriculum/generative-ai": true,
+  "/curriculum/csp": true,
+  "/ai/pl/101": true,
+  "/educate/weblab": true,
+  "/parents": true,
+  "/resources/videos": true,
+  "/curriculum/computer-science-discoveries": true,
+  "/students/careers-in-computer-science": true,
+  "/artificial-intelligence/teaching-assistant": true,
+  "/curriculum/coding-with-artificial-intelligence": true,
+  "/curriculum/how-artificial-intelligence-works": true,
+  "/about/diversity": true,
+  "/curriculum/exploring-generative-artificial-intelligence": true,
+  "/curriculum/computer-science-principles": true,
+  "/professional-learning/artificial-intelligence-101": true,
+  "/tools/web-lab": true,
+  "/en-US/athome": true,
+  "/en-US/educate/resources/videos": true,
+  "/en-US/curriculum/csd": true,
+  "/en-US/careers-with-cs/index": true,
+  "/en-US/curriculum/middle-school": true,
+  "/en-US/ai/teaching-assistant": true,
+  "/en-US/about/careers": true,
+  "/en-US/curriculum/coding-with-ai": true,
+  "/en-US/curriculum/game-design": true,
+  "/en-US/curriculum/how-ai-works": true,
+  "/en-US/diversity": true,
+  "/en-US/curriculum/generative-ai": true,
+  "/en-US/curriculum/csp": true,
+  "/en-US/ai/pl/101": true,
+  "/en-US/educate/weblab": true,
+  "/en-US/parents": true,
+  "/en-US/resources/videos": true,
+  "/en-US/curriculum/computer-science-discoveries": true,
+  "/en-US/students/careers-in-computer-science": true,
+  "/en-US/artificial-intelligence/teaching-assistant": true,
+  "/en-US/curriculum/coding-with-artificial-intelligence": true,
+  "/en-US/curriculum/how-artificial-intelligence-works": true,
+  "/en-US/about/diversity": true,
+  "/en-US/curriculum/exploring-generative-artificial-intelligence": true,
+  "/en-US/curriculum/computer-science-principles": true,
+  "/en-US/professional-learning/artificial-intelligence-101": true,
+  "/en-US/tools/web-lab": true,
+  // P-0-3
+  "/student/middle-high": true,
+  "/student/elementary": true,
+  "/about/hear-from-us": true,
+  "/students": false, //Feature: Header navigation bar - Signed out
+  "/curriculum/elementary-school": true,
+  "/about": false,
+  "/educate/gamelab": true,
+  "/tools/applab": true,
+  "/promote": false,
+  "/curriculum/csf": true,
+  "/educate/spritelab": true,
+  "/about/supporters": true,
+  "/en-US/student/middle-high": true,
+  "/en-US/student/elementary": true,
+  "/en-US/about/hear-from-us": true,
```

Comment:
remove line - not moving the page. just using a redirect

Reply:
write but /en-US/about/hear-from-us is not a redirect just /about/hear-from-us

Reply:
I'm guessing it doesn't hurt to have it here. Just calling out the redirects with /en-US/ in front give sad bee

Reply:
Oh I see, I think the issue is that I'm just blindly adding the locale in front of locale-less paths, this has the unintended effect of creating extraneous entries which "404s" (but they'd 404 on Pegasus too). I did this because I'd rather be safe to cover all the paths and the unintended consequence of having 404s is OK because they were going to 404 anyway. So I think the net effect is that we have extra entries that lead to nowhere.

Reply:
Safe sounds good to me